### PR TITLE
Further updates to Windows installation instructions

### PR DESCRIPTION
**Pull request by jerryg2003**
TBD: Fix missing vertical and horizontal scroll bars (need info from @silverailscolo)

### CODE REVIEW

**Review by bobjacobsen — APPROVED** [review]

**Comment by bobjacobsen** [comment]
Not a windows user, but it looks OK to me.

File: install/WindowsNew.shtml [hunk 1]
```diff
@@ -171,60 +175,77 @@ for earlier versions (2012 and before), <a href="Windows.shtml">refer to the old
...
+                                                                         
+                                        
 
   <li><em><a id="update"></a>Run the JMRI installer</em>
   <p>When the JMRI download is ready, you will be asked to either save it to disk or directly run the installer.</p>
     <ol type="i">
       <li>Installer start-up
-        <p><img src="images/WinInstall02.png" height="489" width="597" alt="installer welcome page"></p></li>
+                                                                                                                                                                                                                                                              
+        <p style="text-align:center;"><img src="images/Windows10-4995-Setup.jpg" alt="installer welcome page"></p></li>
```
Comment (dsand47) [inline install/WindowsNew.shtml:201]: This image is missing.

File: install/WindowsNew.shtml [hunk 2]
```diff
@@ -171,60 +175,77 @@ for earlier versions (2012 and before), <a href="Windows.shtml">refer to the old
...
       <li>Check for Java
-        <p>It will check for a suitable version of Java on your computer and, if required, will connect to the internet to download the most current version for your computer (for an offline installation, refer to the <a href="#offline">Offline installation notes</a> below).
-        <br><img src="images/WinInstall-06a_NoJavaAskDownload.png" height="393" width="503" alt="installer ask Java download"></p>
-        <p>Downloading java
-        <br><img src="images/WinInstall-06b_StartJavaDownload.png" height="393" width="503" alt="installer start Java download"></p></li>
-
+        <p>The installer will check for a suitable version of Java on your computer and, if required, will
+        alert you to which version of Java should be installed.  For example, although these messages may 
+        change based on the version of JMRI you are attempting to install.</p>
+        <p style="text-align:center;"><img src="images/426-NoJavaFound.jpg" alt="No Java found - suggest Java 8"></p>
```
Comment (dsand47) [inline install/WindowsNew.shtml:229]: Image missing.

File: install/WindowsNew.shtml [hunk 3]
```diff
@@ -171,60 +175,77 @@ for earlier versions (2012 and before), <a href="Windows.shtml">refer to the old
...
+        <p style="text-align:center;"><img src="images/4995-NoJavaFound.jpg" alt="No Java found - suggest Java 11"></p>
```
Comment (dsand47) [inline install/WindowsNew.shtml:230]: Image missing.

**Review by dsand47 — CHANGES_REQUESTED** [review]

File: install/WindowsNew.shtml (hunk 1 above)
Comment (jerryg2003) [reply install/WindowsNew.shtml:201]: Sorry about that...

**Review by dsand47 — APPROVED** [review]
Minor oversight:  install/images/Windows10-4995-Complete.jpg not referenced by the html.

**Comment by jerryg2003** [comment]
> Minor oversight: install/images/Windows10-4995-Complete.jpg not referenced by the html.

Yeah, saw that.  Thanks. Will get it in the next Commit (when I figure out fix to be scrolling).

**Review by silverailscolo — APPROVED** [review]
Clean tags, thanks

**Comment by jerryg2003** [comment]
@bobjacobsen @silverailscolo Can you merge and we can see what happens with the scrolling?  Thanks.

**Comment by silverailscolo** [comment]
The scroll bar remains hidden. I will look for a fix embedding the long line of code "WARN - Could not open/create prefs...".

**Pull request merged**